python/cuspatial/cuspatial/tests/test_align.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 NVIDIA CORPORATION.
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+from shapely.geometry import Polygon
+
+import cuspatial
+
+
+def test_align():
+    gpdpdf = gpd.GeoSeries(
+        [
+            Polygon(((-8, -8), (-8, 8), (8, 8), (8, -8))),
+            Polygon(((-2, -2), (-2, 2), (2, 2), (2, -2))),
+        ]
+    )
+    gpdshort = gpdpdf.iloc[0:1]
+    pdf = cuspatial.from_geopandas(gpdpdf)
+    short = cuspatial.from_geopandas(gpdshort)
+    gpdaligned = gpdshort.align(gpdpdf)
+    shortaligned = short.align(pdf)
+    pd.testing.assert_series_equal(gpdaligned[0], shortaligned[0].to_pandas())
```

Comment:
I think it's probably doing the same thing.. But using the [test util from geopandas](https://geopandas.org/en/stable/docs/reference/api/geopandas.testing.assert_geodataframe_equal.html) might be better since it could handle corner cases specific to geographic use cases.

Reply:
I'll try it.